tests/foreman/ui/test_host.py

```diff
@@ -367,6 +367,154 @@ def test_positive_create_libvirt(self):
             self.assertIsNotNone(search)
 
     @run_only_on('sat')
+    @stubbed
+    @tier3
+    def test_positive_create_baremetal_bios(self):
```

Comment:
We do create the hosts using `New Host` dialog in UI. This is same as with `test_positive_create_libvirt`. If nothing else, we want to test the `pxe loader` dropdown from the dialog at least